packages/web/src/components/userprofile/UserProfileLayout.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Heading, HStack, Spacer, useTheme, VStack, Text, Box } from '@chakra-ui/react';
+
+export interface UserProfileData {
+  name: string;
+  pronouns?: string;
+  location?: string;
+  hireable: boolean;
+  purpose: string;
+  schoolName?: string;
+  major?: string;
+  graduationDate?: Date;
+}
+
+export const UserProfileLayout: React.FC<UserProfileData> = (user: UserProfileData) => {
+  const theme = useTheme();
+
+  // Currently only showing the "non-sensitive fields" (see users API)
```

Comment:
- This file should probably be `/components/UserProfile/UserProfile.tsx` since this isn't a layout.
  - A layout will wrap the children passed into it
- To easily expand the props in the future, we probably want to allow our page to pass all the user values in a single `user` property
- We don't need to add all the props into here, this component only cares about what it's displaying, so we only need those. If more fields become available or we want to show more, then we can add them later.

```suggestion
export interface UserProfileProps {
  user: {
    name: string;
    pronouns?: string;
    schoolName?: string;
  }
}

export const UserProfile: React.FC<UserProfileProps> = (user: UserProfileData) => {
```